.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+[target.'cfg(feature = "cargo-clippy")']
```

Comment:
Can we have a comment here explaining that `cargo-clippy` is some kind of pseudo feature?

Reply:
Included and have indicated how to run a standard clippy.